build-logic/convention/src/main/kotlin/GravatarComposeConventionPlugin.kt

```diff
@@ -19,11 +20,39 @@ class GravatarComposeConventionPlugin : Plugin<Project> {
 }
 
 internal fun Project.configureCompose(commonExtension: CommonExtension<*, *, *, *, *, *>) {
+    apply<RoborazziPlugin>()
     apply<ComposeCompilerGradleSubplugin>()
     commonExtension.apply {
         buildFeatures {
             compose = true
         }
+        testOptions {
+            unitTests {
+                // For Roborazzi
+                isIncludeAndroidResources = true
+                all {
+                    // -Pscreenshot to filter screenshot tests
+                    it.useJUnit {
+                        try {
+                            val screenshotTestClass = Class.forName(
+                                "com.gravatar.app.testUtils.roborazzi.ScreenshotTests"
+                            )
+                            if (project.hasProperty("screenshot")) {
+                                includeCategories(screenshotTestClass.name)
+                            } else {
+                                excludeCategories(screenshotTestClass.name)
+                            }
+                        } catch (_: ClassNotFoundException) {
+                            // ScreenshotTests class not found, skip configuration
+                            project.logger.info(
+                                "ScreenshotTests class not found, skipping screenshot test configuration"
+                            )
```

Comment:
❓  I'm a bit confused about what this does. I did update the config to print started tests and when I run:
- `./gradlew testDebug` I could see both screenshots tests being started
- `./gradlew verifyRoborazziDebug -Pscreenshot` I can see other tests being started. `AppModuleTest` and another fake unit test that I've added just to test it.

Am I missing something about how this config should work?

Reply:
Great catch @AdamGrzybkowski!

It wasn't working as I expected. More concretely:

```kotlin
val screenshotTestClass = Class.forName(
       "com.gravatar.app.testUtils.roborazzi.ScreenshotTests"
) 
``` 

My original idea was not to require all `Compose` modules to require the `testUtils`; however, I haven't found another straightforward approach. Also, I would say that we'll need screenshots in all modules defining composable anyway.

I've modified the approach and added the `testUtils` to the `app` module. This should work as expected.